interface/forms/soap/templates/general_new.php

```diff
@@ -68,8 +68,8 @@
 			</table>
 			<div>
 				<!-- Save/Cancel buttons -->
-				<input type="button" id="save" value="<?php echo xla('Save'); ?>"> &nbsp;
-				<input type="button" id="dontsave" class="deleter" value="<?php echo xla('Cancel'); ?>"> &nbsp;
+				<input type="button" id="save" value="<?php echo xlta('Save'); ?>"> &nbsp;
```

Comment:
How did you test this change?

The function xlta does not exist.

xla is the correct